algorithms/sorting.md

```diff
@@ -8,6 +8,13 @@
 - [Recursion](/recursion/recursion.md)
 - [Searching](/algorithms/searching.md) (Particularly Binary Search)
 
+### Motivation
+
+Sorting is important in programming for the same reason it is important in everyday life. It is easier and faster to locate items in a sorted list than unsorted. Sorting algorithms can be used in a program to sort an array for later searching or writing out to an ordered file or report.
```

Comment:
https://www.quora.com/Why-is-sorting-important-in-computer-science-and-programming